x-pack/plugins/security_solution/public/flyout/document_details/right/footer.tsx

```diff
@@ -45,16 +47,24 @@ export const PanelFooter: FC = () => {
   );
 
   return (
-    <FlyoutFooter
-      detailsData={dataFormattedForFieldBrowser}
-      detailsEcsData={dataAsNestedObject}
-      handleOnEventClosed={closeFlyout}
-      isHostIsolationPanelOpen={isHostIsolationPanelOpen}
-      isReadOnly={false}
-      loadingEventDetails={false}
-      onAddIsolationStatusClick={showHostIsolationPanelCallback}
-      scopeId={scopeId}
-      refetchFlyoutData={refetchFlyoutData}
-    />
+    <EuiPanel
```

Comment:
Added `FlyoutFooter`. good catch on dark mode 
![image](https://github.com/elastic/kibana/assets/18648970/d810458d-0a02-4cc7-8513-2cf41bbb75ff)

Reply:
Nice! What do you think about extracting the component to make it usable by other developers?

Reply:
we have a `FlyoutFooter` component that other developer can use [here](https://github.com/elastic/kibana/pull/170279/files#diff-81f00c3cf4e677658889914701825b61ab6c1b99a7a79a8d61c921a11e3d3ea9R19) that is equivalent of
```
<EuiFlyout>
   <EuiPanel padding...>
```
The reason we have a separate wrapper here is because the `FlyoutFooter` [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/footer.tsx#L145) is already a `EuiFlyoutFooter`. Not ideal but i think this is specific to document details for now?
```
<EuiPanel>
   <FlyoutFooter> // from event details
```

Reply:
ah I see that makes sense. I think that's ok then. When we remove the old flyout's code entirely, we can properly bring only what's necessary here and use our own `FlyoutFooter` component 👍